### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,10 +80,15 @@ Earlier versions may also work, but come with no guarantees.
 - Firefox 32+
 - Chrome 38+
 
+Note: "requirejs" must be included in a global context for Comm targets.
```
Comment (willingc) [inline README.md]: Perhaps add example command to do so: ``npm install -g requirejs``

**Comment by willingc** [comment]
@blink1073 Looks great. One small suggestion.

**Comment by blink1073** [comment]
Thanks for the nudge @willingc, updated.

**Comment by jasongrout** [comment]
FYI for those reading this issue, the `utils.loadObject` call is the one that needs requirejs: https://github.com/jupyter/jupyter-js-services/blob/41de1089edf0bb7f75e2a70a5a53bf93771525d4/src/kernel.ts#L976

**Comment by jasongrout** [comment]
Looks like the examples work. Thanks!

**Comment by willingc** [comment]
Thanks @blink1073.

**Pull request merged**